harfbuzz-sys/build.rs

```diff
@@ -1,17 +1,15 @@
+#[cfg(feature = "use-pkg-config-harfbuzz")]
+fn find_harfbuzz_via_pkgconfig() {
+    pkg_config::probe_library("harfbuzz").is_ok();
```

Comment:
This function just calls a single function. I think it could just be inline.

Reply:
True. My first version had more here than just this and I didn't think to revise it when I eliminated the other code.